### ISSUE TITLE
Compressed file sizes seem to be missing on v1.0

### ISSUE THREAD

**Issue by lbebber** [simonihmig/ember-cli-bundle-analyzer#297]
The compressed file sizes seem to be missing on the new version. Is this intentional? Is this an issue on my side? At any rate, there are no clear instructions for enabling it on the main `readme.md` file.

**Comment by simonihmig** [comment]
Right. It is now using https://github.com/danvk/source-map-explorer, and while that has _some_ support for showing gzipped sizes, that is not very good. gzip sizes are not really easy to do, as there is no definite compressed size for a single module, because only the whole bundle gets compressed, and you would need to somehow map that back to individual modules, which there is no "right" way to do. The way source-map-explorer was doing that was confusing me too much to include that.

We should at least document that for sure, and maybe provide an opt-in config?

**Comment by NullVoxPopuli** [comment]
how do you pass options directly to source-map-explorer?
